test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/MaxRequestBufferSizeTests.cs

```diff
@@ -55,7 +55,7 @@ public static IEnumerable<object[]> LargeUploadData
                     Tuple.Create((long?)_dataLength - 1, false),
 
                     // Buffer is exactly the same size as data.  Exposed race condition where
-                    // IConnectionControl.Resume() was called after socket was disconnected.
+                    // ITimeoutControl.Resume() was called after socket was disconnected.
```

Comment:
Oh yeah, that's old, didn't realize that at first. We can change that to "the connection was resumed after socket was disconnected."